common/src/main/java/com/wynntils/core/net/NetResult.java

```diff
@@ -103,7 +102,7 @@ private void doHandle(Consumer<InputStream> onCompletion, Consumer<Throwable> on
 
     private Consumer<InputStream> wrappingHandler(Consumer<InputStream> handler, Consumer<Throwable> onError) {
         return (inputStream) -> {
-            try {
+            try (inputStream) {
                 // FIXME: This is needed for patching class loading issue with Forge EventBus:
```

Comment:
Nonono! This is a breaking change. This was deliberately not closed in the method. I thought this was commented, but it might have been where it was called instead.